src/bilder/images/edxapp/templates/edxapp/xpro/common_values.yml

```diff
@@ -0,0 +1,577 @@
+POLICY_CHANGE_GRADES_ROUTING_KEY: edx.lms.core.default
+PRESS_EMAIL: support@xpro.mit.edu # MODIFIED
+PROCTORING_BACKENDS:
+    DEFAULT: 'proctortrack'
+    'proctortrack':
+      {{ with secret "secret-xpro/edxapp" }}
+      'client_id': '{{ .Data.proctortrack_client_id }}'
+      'client_secret': '{{ .Data.proctortrack_client_secret }}'
+      {{ end }}
+      'base_url': '{{ key "edxapp/proctortrack-base-url" }}'
+    'null': {}
+
+PROCTORING_SETTINGS: {}
+REGISTRATION_EXTRA_FIELDS:
+    city: hidden
+    confirm_email: hidden
+    country: hidden
+    gender: optional
+    goals: optional
+    honor_code: hidden
+    level_of_education: optional
+    mailing_address: hidden
+    terms_of_service: hidden
+    year_of_birth: optional
+RETIRED_EMAIL_DOMAIN: retired.invalid
+RETIRED_EMAIL_PREFIX: retired__user_
+RETIRED_USERNAME_PREFIX: retired__user_
+RETIRED_USER_SALTS:
+- OVERRIDE ME WITH A RANDOM VALUE
+- ROTATE SALTS BY APPENDING NEW VALUES
+RETIREMENT_SERVICE_WORKER_USERNAME: retirement_worker
+RETIREMENT_STATES:
+- PENDING
+- ERRORED
+- ABORTED
+- COMPLETE
+SEGMENT_KEY: null
+SENTRY_ENVIRONMENT: {{ env "ENVIRONMENT" }}
+SERVER_EMAIL: odl-devops@mit.edu # MODIFIED
+# Removing the session cookie domain as it is no longer needed for sharing the cookie
+# between LMS and Studio (TMM 2021-10-22)
+# SESSION_COOKIE_DOMAIN: {{ key "edxapp/cookie-domain" }} # MODIFIED
+SESSION_COOKIE_SECURE: true # MODIFIED
+SESSION_SAVE_EVERY_REQUEST: false
+SITE_NAME: MIT xPRO  # MODIFIED
+SOCIAL_MEDIA_FOOTER_URLS: {}
+SOCIAL_SHARING_SETTINGS:
+    CERTIFICATE_FACEBOOK: false
+    CERTIFICATE_TWITTER: false
+    CUSTOM_COURSE_URLS: false
+    DASHBOARD_FACEBOOK: false
+    DASHBOARD_TWITTER: false
+STATIC_ROOT_BASE: /edx/var/edxapp/staticfiles
+STATIC_URL_BASE: /static/
+STUDIO_NAME: Studio
+STUDIO_SHORT_NAME: Studio
+SUPPORT_SITE_LINK: https://mitx-micromasters.zendesk.com/hc/en-us/requests/new
+SYSTEM_WIDE_ROLE_CLASSES: []
+TECH_SUPPORT_EMAIL: odl-devops@mit.edu  # MODIFIED
+TIME_ZONE: America/New_York
+UNIVERSITY_EMAIL: {{ key "edxapp/sender-email-address" }} # MODIFIED
+USERNAME_REPLACEMENT_WORKER: OVERRIDE THIS WITH A VALID USERNAME
+VIDEO_IMAGE_MAX_AGE: 31536000
+VIDEO_IMAGE_SETTINGS:
+    DIRECTORY_PREFIX: video-images/
+    STORAGE_KWARGS:
+        base_url: /media/
+        location: media/  # MODIFIED
+        bucket: {{ key "edxapp/s3-storage-bucket" }} # MODIFIED
+    VIDEO_IMAGE_MAX_BYTES: 2097152
+    VIDEO_IMAGE_MIN_BYTES: 2048
+VIDEO_TRANSCRIPTS_MAX_AGE: 31536000
+VIDEO_TRANSCRIPTS_SETTINGS:
+    DIRECTORY_PREFIX: video-transcripts/
+    STORAGE_KWARGS:
+        base_url: /media/
+        location: media/  # MODIFIED
+        bucket: {{ key "edxapp/s3-storage-bucket" }} # MODIFIED
+    VIDEO_TRANSCRIPTS_MAX_BYTES: 3145728
+VIDEO_UPLOAD_PIPELINE:
+    BUCKET: ''
+    ROOT_PATH: ''
+WIKI_ENABLED: true
+XBLOCK_FS_STORAGE_BUCKET: null
+XBLOCK_FS_STORAGE_PREFIX: null
+XBLOCK_SETTINGS: {}
+X_FRAME_OPTIONS: DENY
+YOUTUBE_API_KEY: PUT_YOUR_API_KEY_HERE
```

Comment:
We get frequent errors from YouTube about API access. Maybe they'll go way if we omit this, or leave it as null or empty?

Reply:
I'll set it as null